pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -683,7 +685,7 @@ func getPidFromPidFile(pidFile string) (int, error) {
 	}
 	defer file.Close()
 
-	data, err := ioutil.ReadAll(file)
+	data, err := sysutils.ReadAllWithLimit(file, maxReadLength)
```

Comment:
this expects the only file contents to be the PID, so 10MB is overkill. 1K is overkill, but simple.

Reply:
Fixed